frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/common/SlideInOptions.java

```diff
@@ -0,0 +1,23 @@
+package com.virtuslab.gitmachete.frontend.actions.common;
+
+import org.checkerframework.checker.tainting.qual.Untainted;
+
+public class SlideInOptions {
```

Comment:
Okay so now it's compatible with Kotlin? out of curiosity, what were the problems when using Lombok?

Reply:
Yeah, TBH I still don't get the order of how Kotlin and Java compilers interact... looks a bit like a chicken-and-egg problem to me:

* if you compile Java first (in the given Gradle module), then how can you access Kotlin classes from Java code?
* if you compile Kotlin first (in the given Gradle module), then how can you access Java classes from Kotlin code?

While we're at this... lemme summon @Kordyjan as the Kotlin expert! can you shed some light?

Reply:
Okay, looks that Kotlin compiler first parses Java sources without actually compiling them, just to see what's out there in Java code... and then it's easy, coz Java compiler already sees compiled Kotlin classes

Reply:
from what I found out the only solution is to create compilation order - first java, and later kotlin. But in this case we can't use kotlin code from java